multichain-testing/test/kitchen-sink.test.ts

```diff
@@ -152,16 +152,10 @@ const chainConfigs = {
 };
 
 for (const [chainName, config] of Object.entries(chainConfigs)) {
-  for (const wallet of accounts) {
-    const scenario = {
-      ...config,
-      contractName: 'kitchenSink',
-      wallet,
-    };
-    test.serial(
-      `Create account on ${chainName} for ${wallet}`,
-      makeAccountScenario,
-      scenario,
-    );
-  }
+  const scenario = {
+    ...config,
+    contractName: 'kitchenSink',
+    wallet: accounts[0],
+  };
+  test.serial(`Create account on ${chainName}`, makeAccountScenario, scenario);
```

Comment:
consider using the `test.macro`'s title maker and just passing the literals needed to lookup the rest. Then you don't need a loop. e.g.
```
test.serial(makeAccountScenario, 'agoric');
test.serial(makeAccountScenario, 'cosmoshub');
test.serial(makeAccountScenario, 'osmosis');
```

Reply:
Thanks for the push to refactor the test. It led to simplifying the contract!